package.json

```diff
@@ -3,17 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/register": "^7.11.5",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "bootstrap": "^4.5.2",
+    "express": "^4.17.1",
+    "ignore-styles": "^5.0.1",
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts build && node ./server",
+    "dev": "nodemon --ignore 'build/*' --exec \"yarn start\"",
```

Comment:
After running `npm run dev` the yarn start is starting and restarting again and again in the loop. Please install the latest dependencies check on your local update me. I couldn't test it using `npm run dev` after installing the dependencies. I have some changes in the `package.json` and `package-lock.json`

Reply:
Hey @jvsinghk I will take a look as soon as possible :)

Reply:
Hey, @setheal Sure thanks :)

Reply:
Hey, @setheal any updates?

Reply:
Hey @jvsinghk, I checked on my side for your issue with `npm run dev` and I can't replicate it. Can you post your node version please ?

Reply:
It's `12.18.3`. I am updating dependencies from my side in #15 PR. Once it gets merged, please fetch master.

Reply:
Okay, I'll wait for you and try with your version :)

Reply:
I have updated and merged the changes. Please don't use `npm` from now on. Just run `yarn` and `yarn start` after fetching master and deleting `node_modules`. I have removed `package-lock.json` hope it doesn't make any breaking changes.

Reply:
Merged pr, updated node to `12.19.0`. Can you post your node version? I hope now it will work.

Reply:
Hey @setheal, it's done. your wait is over :)